### ISSUE TITLE
Peer Evaluation From Group 106

### ISSUE THREAD

**Issue by aj3616** [krishnak30/US_elections#7]
Opening statement summary:
Hi members of group 9, I am very glad to have the opportunity to review your paper. I think this is a well-written paper, and it is clear that you are attempt to forecast the 2024 U.S. presidential election using polling data. The structure is coherent, and the introduction is engaging. You have a result clearly shown in the first page and it is a nicely down first draft!

Strong positive points:
1. Very clear title, subtitle, abstract
I think your title, subtile and abstract were well written. It fulfills the requirement of showing the result clearly but having an increasing degree of information. We can clearly see which candidate, and what percentage of estimation this paper analyzed to, which answers the main title questions of forecasting 2024 US election. The method used was also clearly explained in abstract.

2. The data was well described and graphs and tables were used
I really liked how you guys clean your data to only leave the variables that you think were important left for further analysis and model building. From the data section, I feel that you guys have a clear idea in mind of a potential model. Other than that, the way you explained each variable and using suitable graphs or tables clearly explain to the readers what each variable is like in the data set.

4. The model is clearly described and a result was estimated by the model
You guys make a compelling case for excluding collinear variables like numeric_grade and pollscore to prevent multicollinearity, which is crucial for obtaining reliable estimates. The use of a multi-step modeling strategy, gradually increasing complexity from Models 1 to 3, is well-justified, and the selection of Model 3 is supported by statistical evidence (higher R^2, lower AIC and BIC values). The presentation of specific state and pollster effects, such as California and Redfield & Wilton Strategies, provides a concrete understanding of how various factors influence support for Kamala Harris.

5. Well constructed and written appendices
The first writing appendix was well constructed with external research and citations of sources used. The second appendix includes a link to the sample survey as well as a glance of waht is included in the survey.

Critical improvements needed:
May consider cite the tools used to generate tables and graphs, now only the janitor package was cited for cleaning data. May also consider modify the overall aesthetics for final polishing. May consider breaking long paragraphs to smaller chunks to keep readers engaged. May consider using \newpage to avoid page breaking in the middle of a table or a complete chunk of information.

Suggestions for improvement:
Move some detailed analysis and the process of developing model to appendix to keep more readers engaged and those who are  still interested can go to appendix to check more for how you come up with the model.

Please consider adding/changing/removing:
1. change the size of the sample survey for not taking too much pages
2. cite r packages for those used to genreate graphs and tables
3. adding more information in the README file
4. R is unavailable in the reference list
5. Introduction can be a little more informative
6. add captions for graphs

Evaluation:
This is a very well written and complete paper that successfully forecast the 2024 US election! Only some minor detail needs some polishing.

Estimated mark:
R is appropriately cited: 0/1
In data section, it clearly said that R was used for cleaning and analyzing but R is not present in the reference list, can take a closer look for that.
Data are appropriately cited: 1/1
Mentioned throughout the paper and present in the reference list!
Class paper: 1/1
no evidence that this is a class paper
LLM usage is documented: 1/1
Documented all the chat and clearly explained in readme.
Title: 2/2
informative and brief
Author, date, and repo: 2/2
all present
Abstract: 4/4
well written with result and how to obtain the result
Introduction: 4/4
Have all the components required
Estimand: 0/1
did not talk abnout estimand explicitly
Data: 10/10
well selected, explained and well used graphs and tables
Measurement: 4/4
clear measurement section explaining what to measure and how
Model: 10/10
well developed model that can successfully give us our result of interest
Results: 10/10 
Discussion: 10/10
The result from the modelling was well discussed
Prose: 6/6
does not see obvious grammar issues and the word usage is informative and official for a paper
Cross-references: 1/1
cross-reference of difference tables and sections are present
Captions: 1/2
have captions for tables not graphs
Graphs/tables/etc: 4/4
clear graphs and tables that are self explanatory
Idealized methodology: 10/10
An idealized methodology was explained in detail
Idealized survey: 4/4
reasonable survey that can take in useful information
Pollster methodology overview and evaluation: 10/10
method taught was used to do the evaluation
Referencing: 3/4
need some minor fixing 
Commits: 2/2
useful commit messages that documents each step
Sketches: 0/2
was not present in the git repository
Simulation: 4/4
successfully simulates all variable of interest, have the correct ranges
Tests-simulation: 4/4
Tests-actual: 4/4
meaningful tests were conducted to both the simulation data and actual data.
Parquet: 1/1
the cleaned data was saved to a parquet file
Reproducible workflow: 3/4
seeds were used, every step was documented and no absolute directory was used, may consider add the seed used in README
Miscellaneous: 3/3
Excellent!

Total mark: 119/126

Conclusion:
In conclusion, this paper successfully forecasts the 2024 U.S. presidential election outcome for Kamala Harris, using a robust model that accounts for polling biases, variability, and regional factors. The insights gained from this model provide a comprehensive and accurate prediction of her support, demonstrating the power of a data-driven approach to election forecasting. Very well done in the limited time given for this paper!

**Comment by DeniseChang9** [comment]
Thank you for your useful recommendations! We've included them in our analysis : )
